src/olympia/zadmin/tests/test_views.py

```diff
@@ -70,61 +70,28 @@ def test_get_index(self):
         assert response.context['user'].email == 'admin@mozilla.com'
 
     def test_django_index(self):
+        # Can access with full admin.
         url = reverse('admin:index')
         response = self.client.get(url)
         assert response.status_code == 200
 
+        # Redirected because no permissions if not logged in.
         self.client.logout()
         response = self.client.get(url)
         self.assert3xx(response, '/admin/models/login/?'
                                  'next=/en-US/admin/models/')
 
+        # Redirected when logged in without enough permissions.
         user = user_factory(username='staffperson', email='staffperson@m.c')
-        self.grant_permission(user, 'Addons:Edit')
         self.client.login(email='staffperson@m.c')
+        response = self.client.get(url)
         self.assert3xx(response, '/admin/models/login/?'
                                  'next=/en-US/admin/models/')
 
-    def test_django_admin_logout(self):
```

Comment:
not that I care about logging out from django admin, but why the test removal?

Reply:
Probably went a little overboard when removing `TestStaffAdmin`, I'll add it back...